src/cli/api/apihelper.go

```diff
@@ -0,0 +1,137 @@
+package api
+
+import (
+	"bytes"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"autoscaler/models"
+	e "cli/errors"
+	"cli/ui"
+)
+
+const (
+	HealthPath  = "/health"
+	PolicyPath  = "/v1/apps/{appId}/policy"
+	MetricPath  = "/v1/apps/{appId}/metric_histories/{metric_type}"
+	HistoryPath = "/v1/apps/{appId}/scaling_histories"
+)
+
+type APIHelper struct {
+	Endpoint *APIEndpoint
+	Client   *CFClient
+}
+
+func (helper *APIHelper) DoRequest(req *http.Request, host string) (*http.Response, error) {
```

Comment:
According to cf cli plugin api, there is no api to expose skip-ssl-validation. 
https://github.com/cloudfoundry/cli/blob/master/plugin/plugin.go
if do so, we need to grep cf config.json directly, and extract values.

Reply:
I guess we don't need to pick up cf cli skip-ssl-validation setting since autoscaler might be a separate deployment not using the same SSL termination as cf